operators/src/processing/temporal_raster_aggregation.rs

```diff
@@ -141,85 +136,128 @@ where
 {
     type RasterType = P;
 
+    #[allow(clippy::too_many_lines)]
```

Comment:
can you maybe extract the creation of the adapter into a method? would make it more readable also